Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.U2D;
+
+namespace MixedRealityToolkit.Utilities
+{
+    public class AtlasPrefabReference : ScriptableObject
+    {
+        [SerializeField] public List<GameObject> Prefabs;
```

Comment:
does this need to be serialized it it's public?

Reply:
If it does, then it needs fixing again in stabilisation and then merging that fix in.
This PR only replicates what is in Stab now.
Although, I agree, this should follow standards.  I'll fix here then submit another pr to stabilisation

Reply:
Nah, I don't think it's too necessary for stabilization.
Really up to you tho.